lib/oper1.g

```diff
@@ -745,7 +745,7 @@ InstallAttributeFunction(
 ##  <#/GAPDoc>
 ##
 IsPrimeInt := "2b defined";
-
+MakeWriteOnceAtomic := "2b defined";
```

Comment:
On Sat, Mar 21, 2015 at 06:25:58PM -0700, Reimer Behrends wrote:
> > @@ -745,7 +745,7 @@ InstallAttributeFunction(
> >  ##  <#/GAPDoc>
> >  ##
> >  IsPrimeInt := "2b defined";
> > -
> > +MakeWriteOnceAtomic := "2b defined";
> 
> `MakeWriteOnceAtomic()` is a kernel function, so the only reason to
> have it there is for when oper1.g is getting precompiled (maybe not
> even then?). Otherwise, it should already exist and it seems like it
> would be overwritten. Or am I missing something here?
Indeed I added it (not quite my idea) as it wasnt't defined at that
stage.
I also need to add, that on 3rd thoughts I added a race condition...
It should be replaced by some atomic hash-table, or the suggestion, an
atomic record.

Greetings 
 Claus
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/gap-system/gap/pull/108/files#r26898821